backend-shared/util/util.go

```diff
@@ -156,3 +160,17 @@ func GetRESTConfig() (*rest.Config, error) {
 	res.Burst = 250
 	return res, nil
 }
+
```

Comment:
Sorry I didn't think of this during first review, but I think we can write a quick little unit test for this, just so we have some coverage for this function. (It can likely be just testing the empty env var case, and then the non-empty env var case. I'm pretty sure `os.SetEnv` can be used in a test, so no need to mock this)

Reply:
Done